### ISSUE TITLE
Export DefaultTooltipContent and DefaultLegendContent

### ISSUE THREAD

**Issue by Pagebakers** [recharts/recharts#3288]
- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I want to style the tooltip using Chakra UI to make it look the same as the Chakra UI tooltip and keep the rest of the functionality the same.

```tsx
<Tooltip
 content={(props) => {
    return (
        <chakra.div __css={styles}>
         <DefaultTooltipContent {...props} />
        </chakra.div>
    )
}}
```

I can import the component from `import { DefaultTooltipContent } from 'recharts/lib/component/DefaultTooltipContent'`, however this is problematic in ESM projects and visaversa. It would be ideal to be able import this from the root.

I've a PR ready, it's an easy fix to allow for better customization.

### What does the proposed API look like?
Just exporting DefaultTooltipContent and DefaultLegendContent from index.ts

**Comment by nikolasrieble** [comment]
I have had the same issues and agree. It would make sense, especially as it enables wrapping the default Tooltip with custom behaviour - which is an idea that suits recharts very well.

So far I do not see any issues.  

Would you like to do a PR for it? @Pagebakers

**Comment by Pagebakers** [comment]
I have a pr ready, I'll open it in the next days.
Thanks!

**Comment by Pagebakers** [comment]
Looks like I totally forgot to open that PR, thanks @oschwede

**Comment by ckifer** [comment]
Released in 2.7
